### PR TITLE
Change references from Docker to Container.

### PR DESCRIPTION
**Pull request by rhatdan**
Docker container -> container

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ OKD: The Origin Community Distribution of Kubernetes
 * Multi-tenancy support, including team and user isolation of containers, builds, and network communication.
   * Allow developers to run containers securely with fine-grained controls in production.
   * Limit, track, and manage the developers and teams on the platform.
-* Integrated Docker registry, automatic edge load balancing, cluster logging, and integrated metrics.
+* Integrated container registry, automatic edge load balancing, cluster logging, and integrated metrics.
```
Comment (bparees) [inline README.md]: imho it's an image registry, not a container registry.

Comment (rhatdan) [reply README.md]: It might literally be an image registry, but I have never heard a customer call it such.  Perhaps a
container image registry.

Comment (bparees) [reply README.md]: which is probably our fault for describing it badly(and creating something called the "redhat container catalog").  we should stop reinforcing it.

Comment (rhatdan) [reply README.md]: So how do I go forward.  Do you want me to change it to container image registry?

Comment (bparees) [reply README.md]: i guess so. i was hoping some other people would weigh in.

**Comment by rhatdan** [comment]
@bparees PTAL
I remember someone commenting I should not change `api` directory.

**Pull request merged**